lib/ferrum/page/frames.rb

```diff
@@ -75,8 +75,11 @@ def frames_subscribe
           frame_id = params.dig("context", "auxData", "frameId")
 
           unless @main_frame.id
-            @main_frame.id = frame_id
-            @frames[frame_id] = @main_frame
+            root_frame = command('Page.getFrameTree').dig('frameTree', 'frame', 'id')
```

Comment:
I think it's nice to have `public` method for this `Ferrum::Page::Frames#root_frame`

```
def root_frame
  command('Page.getFrameTree').dig('frameTree', 'frame', 'id')
end
```